pkg/rpc/client.go

```diff
@@ -88,9 +87,13 @@ func (c *RPCClient) rpcRequest(ctx context.Context, data io.Reader) ([]byte, err
 	if err != nil {
 		return nil, err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		if err := resp.Body.Close(); err != nil {
+			klog.Errorf("could not close response body: %v", err)
+		}
+	}()
```

Comment:
This would generally only error out if the body was already closed, so the error message is likely not correct. Think of this more as an optimistic "close if not already closed". If it errors it is ok to discard.